Mathlib/Order/Atoms/Finite.lean

```diff
@@ -28,21 +28,25 @@ variable [LE α] [BoundedOrder α] [IsSimpleOrder α]
 
 section DecidableEq
 
-/- It is important that `IsSimpleOrder` is the last type-class argument of this instance,
-so that type-class inference fails quickly if it doesn't apply. -/
-instance (priority := 200) [DecidableEq α] : Fintype α :=
+/-- It is important that `IsSimpleOrder` is the last type-class argument of this instance,
+so that type-class inference fails quickly if it doesn't apply.
+
+Note that as of 2025-08-13, this is false. Could someone investigate? -/
+scoped instance (priority := 200) [DecidableEq α] : Fintype α :=
   Fintype.ofEquiv Bool equivBool.symm
 
 end DecidableEq
 
-instance (priority := 200) : Finite α := by classical infer_instance
+scoped instance (priority := 200) : Finite α := by classical infer_instance
```

Comment:
This one seems more of a shame to lose.

Reply:
Sorry, are you asking that I revert this?

Reply:
I think probably closer to asking for a second opinion from another reviewer

Reply:
It is a bit more of a shame. But it's not clear to me how much.

Let's just try this scoped for a while.